.github/workflows/docs-pr.yml

```diff
@@ -114,3 +114,16 @@ jobs:
           NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
           NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
         timeout-minutes: 1
+
+  add_comment:
+    needs: [deploy_preview]
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Tag dev rel in comment
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          message: |
+            FYI @noir-lang/developerrelations on Noir doc changes.
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Seems like this could be added into the `deploy-message` field in the job above.

Reply:
`deploy-message` is a custom message printed in CI logs (instead of in the PR comment).

Example: https://github.com/noir-lang/noir/actions/runs/7908036275/job/21586777257#step:4:6

Reply:
Ah, so it is. Kinda dumb on the part of the action to have a hardcoded comment.